WordPress/src/main/java/org/wordpress/android/ui/accounts/login/Login2FaFragment.java

```diff
@@ -54,13 +71,46 @@
     ArrayList<Integer> mOldSitesIDs;
 
     private String mEmailAddress;
+    private String mIdToken;
+    private String mNonce;
```

Comment:
It doesn't _have_ to be a global variable.  It was written that way for readability for a couple reasons.
1. To denote the passed nonce is variable since it can be any one of the three nonce types.
2. Since the `setAuthCodeTypeAndNonce` is setting the `mType` based on the code, it makes sense to set the `mNonce` in the same method.

The `setAuthCodeTypeAndNonce` could be changed to something like `setAuthCodeTypeAndReturnNonce`, but I think that would be harder to grok.